packages/document/doc-tools-doc/docs/en/plugin/official-plugins/typedoc.mdx

```diff
@@ -0,0 +1,111 @@
+# typedoc
+
+Integration of [TypeDoc](https://github.com/TypeStrong/typedoc)'s Modern.js Doc Plugin for Automatically Generating API Documentation for TS Modules.
+
+## Installation
+
+```bash
+# npm
+npm install @modern-js/doc-plugin-typedoc
+# yarn
+yarn add @modern-js/doc-plugin-typedoc
+# pnpm
+pnpm install @modern-js/doc-plugin-typedoc
+```
+
+## Usage
+
+```ts
+import { docTools, defineConfig } from '@modern-js/doc-tools';
+import { pluginTypeDoc } from '@edenx/doc-plugin-typedoc';
+import path from 'path';
+
+export default defineConfig({
+  doc: {
+    plugins: [
+      pluginTypeDoc({
```

Comment:
`doc.plugins` and `plugins` may confuse user, we may need to unify the two plugin config in Rspress.

Reply:
In Rspress，we only have plugins config and the modern.js plugin is not included.